packages/n/nemo-extensions/package.yml

```diff
@@ -50,64 +51,6 @@ description:
     - ^nemo-seahorse : Extension for Nemo which allows encryption and decryption of OpenPGP files using GnuPG - the GNU Privacy Guard program.
     - ^nemo-share : Nemo Share allows you to quickly share a folder from the Nemo file manager without requiring root access.
     - ^nemo-terminal : Nemo Terminal is an embedded terminal for Nemo. It embeds a terminal pane into Nemo that is accessible by hotkey (default F4) and automatically follows the currently active directory in Nemo.
-patterns   :
```

Comment:
You're just moving all of `patterns` right? The diff is not helping here, lol

Reply:
Correct